package/MDAnalysis/core/selection.py

```diff
@@ -979,6 +1012,7 @@ def __init__(self, parser, tokens):
             raise ValueError(
                 "Invalid operator : '{0}' Use one of : '{1}'"
                 "".format(oper, self.ops.keys()))
+        self.value = float(value)
```

Comment:
It would be nice as people working in python expect chained comparison to work. It seems slightly out of scope for this PR so, beside my [small comment above](https://github.com/MDAnalysis/mdanalysis/pull/1156#discussion_r95631318), the PR looks good to go. Would you like to have a look at inverted syntax and chained comparisons or shall we open new issues?